### PR TITLE
Update technologies.yaml

### PR DESCRIPTION
**Pull request by sandrinecharousset**
added variables: Variable Cost, Variable Cost with/without emission cost for electricity inputs

### CODE REVIEW

**Comment by sandrinecharousset** [comment]
@loeffko

File: definitions/variable/technology/technologies.yaml [hunk 1]
```diff
@@ -76,13 +76,45 @@
...
-    description: Capital cost for {Fuel}
+    description: Variable cost for {Fuel}
     unit: [EUR_2020/MWh, USD_2010/MWh]
     skip-region-aggregation: true
 - Variable Cost|Heat|Industrial|{Fuel}:
-    description: Capital cost for {Fuel}
+    description: Variable cost for {Fuel}
+    unit: [EUR_2020/MWh, USD_2010/MWh]
+    skip-region-aggregation: true
+- Variable Cost (incl. Fuel Cost)|Electricity|{Electricity Input}:
```
Comment (danielhuppmann) [inline definitions/variable/technology/technologies.yaml:86]: I suggest to instead interpret "Variable Cost" as *only* the non-fuel-related costs and then add a separate variable "Fuel Cost|Electricity|..."

File: definitions/variable/technology/technologies.yaml [hunk 2]
```diff
@@ -76,13 +76,45 @@
...
+    description: Variable cost for {Electricity Input}, including the fuel cost
+    unit: [EUR_2020/MWh, USD_2010/MWh]
+    skip-region-aggregation: true
+- Variable Cost (excl. Fuel Cost)|Electricity|{Electricity Input}:
+    description: Variable cost for {Electricity Input}, without the fuel cost
     unit: [EUR_2020/MWh, USD_2010/MWh]
     skip-region-aggregation: true
+- Levelized Cost|Capex|Electricity|{Electricity Input}:
```
Comment (danielhuppmann) [inline definitions/variable/technology/technologies.yaml:94]: Is this really different from the existing variable "Capital Cost|Electricity|..."?

**Review by danielhuppmann — COMMENTED** [review]
A few questions and suggestions inline.

File: definitions/variable/technology/technologies.yaml (hunk 1 above)
Comment (sandrinecharousset) [reply definitions/variable/technology/technologies.yaml:86]: Well, I am also going to add the variable 'Fuel Cost' but the way of getting the Variable Cost (incl. Fuel Cost) is not at all trivial and this is an output of Genesys-Mod so I would like to keep the Variable Cost (incl. Fuel Cost) as sth that can be retrieved from the platform as it is a required input for some models.
But we can remove the Variable Cost (excl. Fuel Cost) and add in the description of 'Variable Cost' that it is excluding the Fuel cost (which is not trivial and was previously mis-interpreted by more than one modelling team)

Comment (loeffko) [reply definitions/variable/technology/technologies.yaml:86]: I agree with Daniel (and this is also how we have the variable costs natively in our model). So in my understanding, the variable costs are just those associated with the technology itself.

I can provide them separately without an issue, so adding a separate variable sounds good to me!

File: definitions/variable/technology/technologies.yaml (hunk 2 above)
Comment (loeffko) [reply definitions/variable/technology/technologies.yaml:94]: Yes, this is indeed different, as @sandrinecharousset says. For capital costs, we list the actual investment costs in M€/GW, whereas within the levelized cost calculations for the technology, they are essentially converted to a "per energy produced" basis.

Comment (sandrinecharousset) [reply definitions/variable/technology/technologies.yaml:94]: thanks @loeffko for the 2 comments
@danielhuppmann seems we are all ok now, so this PR can be merged, right?

**Review by loeffko — APPROVED** [review]

**Comment by sandrinecharousset** [comment]
Closing this PR as some of the changes were already included and the remaining one (levelized costs) is in a new PR

**Pull request closed without merging**